extensions/eclipse-che-theia-remote-impl-che-server/package.json

```diff
@@ -18,15 +18,15 @@
     "@eclipse-che/workspace-client": "latest"
   },
   "scripts": {
-    "prepare": "yarn clean && yarn build",
+    "prepare": "yarn clean && yarn build && yarn test",
     "clean": "rimraf lib",
     "format": "if-env SKIP_FORMAT=true && echo 'skip format check' || prettier --check '{src,tests}/**/*.ts' package.json",
     "format:fix": "prettier --write '{src,tests}/**/*.ts' package.json",
     "lint": "if-env SKIP_LINT=true && echo 'skip lint check' || eslint --cache=true --no-error-on-unmatched-pattern=true '{src,tests}/**/*.ts'",
     "lint:fix": "eslint --fix --cache=true --no-error-on-unmatched-pattern=true \"{src,tests}/**/*.{ts,tsx}\"",
     "compile": "tsc",
     "build": "concurrently -n \"format,lint,compile\" -c \"red,green,blue,purple\" \"yarn format\" \"yarn lint\" \"yarn compile\"",
-    "test": "if-env SKIP_TEST=true && echo 'skip test' || jest --forceExit",
+    "test": "jest",
```

Comment:
Would it still be possible to skip the tests ?

Reply:
Yep, added the opportunity to skip the tests.